# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1337372164 25200
#      Fri May 18 13:16:04 2012 -0700
# Node ID 389f49a886568799a5b4d87c2f68af60ee264913
# Parent  af953acc055d901ae28c10038aaa06f43e2db12d
NEWS: Re-order list of changes for 3.8.0 to place deprecated functions last.

* NEWS: Re-order list of changes for 3.8.0 to place deprecated functions last.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -54,16 +54,22 @@ Summary of important user-visible change
      c,...
     ] = deal (1,2,3)
 
  ** Warning IDs renamed:
 
     Octave:array-as-scalar => Octave:array-to-scalar
     Octave:array-as-vector => Octave:array-to-vector
 
+ ** The colormap function now provides new options--"list", "register",
+    and "unregister"--to list all available colormap functions, and to
+    add or remove a function name from the list of known colormap
+    functions.  Packages that implement extra colormaps should use these
+    commands with PKG_ADD and PKG_DEL statements.
+
  ** Other new functions added in 3.8.0:
 
       colorcube   splinefit
       lines       tetramesh
       rgbplot     shrinkfaces
       findfigs
 
  ** Deprecated functions.
@@ -84,22 +90,16 @@ Summary of important user-visible change
       <none yet>
 
     The following keywords have been deprecated in Octave 3.8 and will
     be removed from Octave 3.12 (or whatever version is the second major
     release after 3.8):
 
       static
 
- ** The colormap function now provides new options "list", "register",
-    and "unregister" to list all available colormap functions, and to
-    add or remove a function name from teh list of known colormap
-    functions.  Packages that implement extra colormaps should use these
-    commands with PKG_ADD and PKG_DEL statements.
-
 Summary of important user-visible changes for version 3.6:
 ---------------------------------------------------------
 
  ** The PCRE library is now required to build Octave.  If a pre-compiled
     package does not exist for your system, you can find PCRE sources
     at http://www.pcre.org
 
  ** The ARPACK library is no longer distributed with Octave.
